server.js

```diff
@@ -16,9 +20,66 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+// Mongoose model
+const Thought = mongoose.model("Thought", {
+  message: {
+    type: String,
+    required: true,
+    maxlength: 140,
+    minlength: 5,
+  },
+  hearts: { type: Number, default: 0 },
+  createdAt: { type: Date, default: Date.now },
+});
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = expressListEndpoints(app);
+  res.json(endpoints);
+});
+
+app.get("/thoughts", async (req, res) => {
+  // Get all the thoughts
+  const thoughts = await Thought.find().sort(
+    { createdAt: "desc" }).limit(20).exec();
```

Comment:
I might be completely wrong here, but I just reacted because I did it differently. If there are errors thrown by "await Thought.find()", would they be handled? I'm thinking since it's outside of the try/catch?

Reply:
I'd also put it in the try block 👍